pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -125,31 +127,36 @@ func Run(ctx context.Context, kbClient kbclient.Client, namespace string, waitTo
 	}
 
 	// CRDs
-	veleroLabels := labels.FormatLabels(install.Labels())
-	crdList := apiextv1beta1.CustomResourceDefinitionList{}
-	opts := kbclient.ListOptions{
-		Namespace: namespace,
-		Raw: &metav1.ListOptions{
-			LabelSelector: veleroLabels,
+
+	veleroLabelSelector := labels.SelectorFromSet(install.Labels())
+	opts := []kbclient.DeleteAllOfOption{
+		kbclient.InNamespace(namespace),
+		kbclient.MatchingLabelsSelector{
+			Selector: veleroLabelSelector,
 		},
 	}
-	if err := kbClient.List(context.Background(), &crdList, &opts); err != nil {
-		errs = append(errs, errors.WithStack(err))
-	} else {
-		if len(crdList.Items) == 0 {
-			fmt.Print("Velero CRDs do not exist, skipping.\n")
+	v1CRDsRemoved := false
+	v1crd := &apiextv1.CustomResourceDefinition{}
+	if err := kbClient.DeleteAllOf(ctx, v1crd, opts...); err != nil {
+		if meta.IsNoMatchError(err) {
+			fmt.Print("V1 Velero CRDs not found, skipping...")
 		} else {
-			veleroLabelSelector := labels.SelectorFromSet(install.Labels())
-			opts := []kbclient.DeleteAllOfOption{
-				kbclient.InNamespace(namespace),
-				kbclient.MatchingLabelsSelector{
-					Selector: veleroLabelSelector,
-				},
-			}
-			crd := &apiextv1beta1.CustomResourceDefinition{}
-			if err := kbClient.DeleteAllOf(ctx, crd, opts...); err != nil {
-				errs = append(errs, errors.WithStack(err))
+			errs = append(errs, errors.WithStack(err))
+		}
+	} else {
+		v1CRDsRemoved = true
+	}
+
+	// Remove any old V1Beta1 CRDs hanging around.
```

Comment:
This chunk will be removed in some future release right?

Reply:
```suggestion
// Remove any old Velero v1beta1 CRDs hanging around.
```